Gemfile

```diff
@@ -12,6 +12,10 @@ gem 'ransack'
 gem 'faker'
 gem 'as-duration'
 gem 'kaminari'
+gem 'carrierwave'
+gem 'fog'
+gem 'dotenv-rails'
```

Comment:
I think we only want dotenv-rails for test and development. We will use rails_12factor for production (ie- no .env files in production)